config/samples/core_v1alpha1_humiopdfrenderservice.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: core.humio.com/v1alpha1
+kind: HumioPdfRenderService
+metadata:
+  name: pdf-render-service
+  namespace: logging
+spec:
+  # Required fields
+  image: humio/pdf-render-service:0.0.60--build-102--sha-c8eb95329236ba5fc65659b83af1d84b4703cb1e
+  # Optional fields
+  # Image pull secrets
+  imagePullSecrets:
+    - name: regcred
+  # Number of replicas to run/humio-operator/pkg/humio/humio_tes
+  replicas: 2
+  # Port configuration
+  port: 5123
+  # Service type (matches the Terraform configuration)
+  serviceType: ClusterIP
+  # Environment variables
+  env:
```

Comment:
I think `env` should be `environmentVariables` to remain consistent with this project.